contrib/localnet/scripts/start-zetacored.sh

```diff
@@ -269,6 +269,9 @@ then
 # v2 erc20 revert tester
   address=$(yq -r '.additional_accounts.user_v2_erc20_revert.bech32_address' /root/config.yml)
   zetacored add-genesis-account "$address" 100000000000000000000000000azeta
+# bank precompile user
+  address=$(yq -r '.additional_accounts.user_bank_precompile.bech32_address' /root/config.yml)
+  zetacored add-genesis-account "$address" 100000000000000000000000000azeta
```

Comment:
Are the funds used for anything?

Reply:
No, the bank won't interact with azeta. I'll check if the account can be created with just 0.

Reply:
I confirmed the address can be added with 0 azeta, so updating it.